docs/faq.md

```diff
@@ -0,0 +1,84 @@
+# Frequently Asked Questions
+
+The present document lists a couple of frequently asked questions from our users and adopters.
+
+## How do I use x CNI with cluster?
+
+There is a great chance your tenant cluster CNI will conflict with infrastructure's cluster CNI.
+To avoid issues, you need first to make sure you are using VXLAN on your CNI, and a customized PORT. Below you can find a few examples:
+
+### Calico
+
+To use Calico CNI, you need to set `CALICO_IPV4POOL_IPIP` to `Never` and `CALICO_IPV4POOL_VXLAN` to `Always`. To customize your VXLAN port, you need to set `FELIX_VXLANPORT` to the customized port number.
+
+### Cilium
+
+To use Cilium CNI, make sure your `tunnel` is set to `vxlan`. To customize your VXLAN port, you need to set `tunnel-port` to the customized port number.
+
+### Flannel
+
+To use Flannel you will need to define your `Backend` manually in `net-conf.json` file, usually inside your `ConfigMap`. You must set `Type` to `vxlan` and `Port` to your customized port number, like the example below:
+```
+  net-conf.json: |
+    {
+      "Network": "10.243.0.0/16",
+      "Backend": {
+        "Type": "vxlan",
+        "Port": 1234
+      }
+    }
+```
+
+## Whenever I reboot a VM in Kubevirt, it looses is IP address, how do I fix it?
```

Comment:
looses/loses

also, this shouldn't be a problem. restarting a vm will result in a new ip, but i believe kubernetes is resilient to nodes having a different ip after restart.

does this for sure cause an issue?

Reply:
<img width="505" alt="image" src="https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/assets/41125128/8316d08f-6ce7-45c0-b438-200bdcc257fd">

This is what I was trying to document here.

Thanks

Reply:
ah yes, ha, i remember that now. I had experimented with worker nodes receiving new IP addresses after restart with some success, but it is true that control plane nodes offer a different set of challenges